server/channels/app/export.go

```diff
@@ -163,6 +171,19 @@ func (a *App) BulkExport(ctx request.CTX, writer io.Writer, outPath string, job
 	return nil
 }
 
+func (a *App) exportAttachments(ctx request.CTX, attachments []imports.AttachmentImportData, outPath string, zipWr *zip.Writer, totalExportedFiles *int, totalFiles int) *model.AppError {
+	for _, attachment := range attachments {
+		if err := a.exportFile(outPath, *attachment.Path, zipWr); err != nil {
+			return err
+		}
+		*totalExportedFiles++
+		if *totalExportedFiles%10 == 0 {
+			ctx.Logger().Info("Bulk export: exporting file attachments progress", mlog.Int("total_exported_attachments", *totalExportedFiles), mlog.Int("total_attachments", totalFiles))
```

Comment:
Maybe the log message should include the number of files that still need to get exported?

Reply:
It does, `totalExportedFiles` is the number of successfully export files and `totalFiles` is the total number of files that need to be exported. You would just subtract them to see how many are left.

Maybe I should change the log fields to: `total_successfully_exported_files` and `total_files_to_export`?

Reply:
The naming change makes sense :+1: